addons/udes_stock/models/stock_picking_print_strategy.py

```diff
@@ -23,3 +23,18 @@ def strategies(self, picking):
         return self.search([
             ('picking_type_id', '=', picking.picking_type_id.id),
         ])
+
+    @api.multi
+    def records(self, picking, context=None):
```

Comment:
Given the way you've written the change in odoo-print, `context` will always be a valid dict. You can safely remove the default value from the signature.